browser/search_engines/guest_window_search_engine_provider_service.cc

```diff
@@ -16,8 +17,8 @@
 GuestWindowSearchEngineProviderService::GuestWindowSearchEngineProviderService(
     Profile* otr_profile)
     : SearchEngineProviderService(otr_profile) {
-  DCHECK(IsGuestProfile(otr_profile));
-  DCHECK(!otr_profile->IsTorProfile());
+  DCHECK(brave::IsOTRGuestProfile(otr_profile));
+  DCHECK(!tor::IsTorProfile(otr_profile));
```

Comment:
I don't think this is necessary. If it's a guest profile from the path then it's definitely not a tor profile and this check will be even more irrelevant once Tor profiles are not based on guest profiles

Reply:
I think the original intent (Profile::GetProfileType() == GUEST_PROFILE) here was not that it was a guest profile, but that it was an OTR profile.